tests/unit/data_formats/test_yolo_format.py

```diff
@@ -1221,3 +1230,73 @@ def _make_some_annotation_values():
     )
     def test_can_report_invalid_field_type(self, field, field_name, test_dir):
         self._check_can_report_invalid_field_type(field, field_name, test_dir)
+
+    def test_can_use_sub_labels_hint(self, test_dir, helper_tc):
+        source_dataset = self._prepare_dataset(test_dir)
+        expected_dataset = Dataset.from_iterable(
+            source_dataset,
+            categories={
+                AnnotationType.label: LabelCategories.from_iterable(
+                    [
+                        "test",
+                        ("custom_name", "test"),
+                        ("another_custom_name", "test"),
+                        ("test_name", "test"),
+                        ("42", "test"),
+                    ]
+                ),
+                AnnotationType.points: PointsCategories.from_iterable(
+                    [(0, ["custom_name", "another_custom_name", "test_name", "42"], set())]
+                ),
+            },
+        )
+        parsed_dataset = Dataset.import_from(
+            test_dir,
+            self.IMPORTER.NAME,
+            skeleton_sub_labels={
+                "test": ["custom_name", "another_custom_name", "test_name", "42"],
+            },
+        )
+        compare_datasets(helper_tc, expected_dataset, parsed_dataset)
+
+    def test_fails_on_wrong_number_of_sub_labels_in_hint(self, test_dir):
+        self._prepare_dataset(test_dir)
+        with pytest.raises(
+            InvalidAnnotationError, match="Number of points in skeletons according to config file"
+        ):
+            Dataset.import_from(
+                test_dir,
+                self.IMPORTER.NAME,
+                skeleton_sub_labels={
+                    "test": [
+                        "custom_name",
+                        "another_custom_name",
+                        "test_name",
+                        "42",
+                        "extra_sub_label",
+                    ],
+                },
+            )
+
+    def test_fails_on_lack_of_skeleton_name_in_hint(self, test_dir):
```

Comment:
Please rename tests to use naming convention: `test_can_...`

Reply:
fixed